### ISSUE TITLE
Why is an ./html folder being created?

### ISSUE THREAD

**Issue by natemccurdy** [puppetlabs/puppet_litmus#105]
Running `bundle exec rake litmus:acceptance:parallel` in my module creates an `html/` folder at the root of the module. Why is that and how do I turn that off?

![html folder](https://user-images.githubusercontent.com/10200384/57399532-4b09ca80-7186-11e9-9211-19136a5b2ad6.png)

```
~/src/yumrepos_fact [litmus]$  cat .rspec
--color
--format documentation
```

**Comment by natemccurdy** [comment]
Looks like it's this, but I don't see `html` being set for the format anywhere in my code base: https://coderwall.com/p/gfmeuw/rspec-test-results-in-html

**Comment by tphoney** [comment]
that is the rspec output, 
![image](https://user-images.githubusercontent.com/10402706/57451848-431b5a80-725a-11e9-8086-54cf2c64ad2c.png)

the goal of that was to create artifacts that an engineer could better use, to diagnose any testing failures.

It should be configurable

**Comment by natemccurdy** [comment]
Yeah, that's the one.

And yes, I think that it:
* Should be configurable and/or able to be turned off.
* Should be ignored in `.gitignore`

**Comment by tphoney** [comment]
fix in #107

**Comment by tphoney** [comment]
this has been merged. thanks for the bug @natemccurdy  !
